src/Barcoder/DataMatrix/DataMatrixEncoder.cs

```diff
@@ -6,12 +6,13 @@ namespace Barcoder.DataMatrix
 {
     public static class DataMatrixEncoder
     {
-        public static IBarcode Encode(string content)
+        public static IBarcode Encode(string content, byte fixedNumberOfRows = 0)
```

Comment:
```suggestion
        public static IBarcode Encode(string content, int? fixedNumberOfRows = null)
```

Public API's better use `int` instead of `byte`. Also, a nullable int better defines the 'optional' behavior of this parameter.